Mathlib/Data/ENat/Basic.lean

```diff
@@ -89,6 +89,9 @@ theorem coe_sub (m n : ℕ) : ↑(m - n) = (m - n : ℕ∞) :=
 @[simp] theorem mul_top (hm : m ≠ 0) : m * ⊤ = ⊤ := WithTop.mul_top hm
 @[simp] theorem top_mul (hm : m ≠ 0) : ⊤ * m = ⊤ := WithTop.top_mul hm
 
+theorem top_pow {n : ℕ} (n_pos : 0 < n) : (⊤ : ℕ∞)^n = ⊤ := WithTop.top_pow n_pos
+#align enat.top_pow ENat.top_pow
```

Comment:
ditto

Reply:
The "no #align for new statements" also applies here. That's the meaning of "ditto".

Reply:
No, I did not... sorry. 

Done now.